### ISSUE TITLE
forEach_()/map_() callback invocation issue in streamline-runtime 1.1.3

### ISSUE THREAD

**Issue by keithmo** [Sage/streamlinejs#365]
Sorry to be a pest, but there's a problem with the fix to issue #364. Line 186 of `src/builtins._js` reads:
```
result[k] = fn.call(thisObj, _, this[k], this);
```
It should be:
```
result[k] = fn.call(thisObj, _, this[k], i, this);
```

**Comment by bjouhier** [comment]
Ooops! I'll fix quickly.

**Comment by bjouhier** [comment]
Fixes: `streamline-runtime@1.1.4`
